### PR TITLE
improve debianization

### PR DESCRIPTION
**Pull request by dkarpukhin**

### CODE REVIEW

**Comment by 3Hren** [comment]
ur code is shit!

File: debian/cocaine-isolate-daemon-upstart.upstart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+description     "cocaine isolate daemon"
+
+start on filesystem
+stop on runlevel [06]
+
+expect fork
```
Comment (antmat) [inline debian/cocaine-isolate-daemon-upstart.upstart]: wtf?

File: debian/control [hunk 2]
```diff
@@ -11,3 +11,16 @@ Depends: ${misc:Depends}
 Description: Run your processes, Docker and Porto containers on Cocaine Cloud
  It an external isolation plugin for Cocaine Core. Works with Docker, Porto,
  processes.
+
+Package: cocaine-isolate-daemon-ubic
```
Comment (antmat) [inline debian/control]: wtf?

File: debian/control [hunk 3]
```diff
@@ -11,3 +11,16 @@ Depends: ${misc:Depends}
...
+Architecture: amd64
+Depends: cocaine-isolate-daemon,
+         lsb-base (>= 3.2-14),
+         ubic
+Description: add ubic configuration for run cocaine-isolate-daemon as service
```
Comment (3Hren) [inline debian/control]: Inconsistency with the previous description style: should start with uppercase letter

File: debian/control [hunk 4]
```diff
@@ -11,3 +11,16 @@ Depends: ${misc:Depends}
...
+
+Package: cocaine-isolate-daemon-upstart
+Architecture: amd64
+Depends: cocaine-isolate-daemon
+Description: add upstart configuration for run cocaine-isolate-daemon as service
```
Comment (3Hren) [inline debian/control]: Here too.

File: debian/control (hunk 2 above)
Comment (antmat) [inline debian/control]: Ah, ok, see the install file now

**Comment by antmat** [comment]
no conflict between ubic and upstart packages declared

**Pull request merged**